hs_core/models.py

```diff
@@ -668,6 +668,31 @@ def remove(cls, element_id):
         raise ValidationError("Description element of a resource can't be deleted.")
 
 
+class Citation(AbstractMetaDataElement):
+    """Define Citation metadata element model."""
+
+    term = 'Citation'
+    value = models.CharField(max_length=750)
+
+    def __unicode__(self):
+        """Return value field for unicode representation."""
+        return self.value
+
+    class Meta:
```

Comment:
Everything below can be removed, it's just duplicating the functionality in the parent.

Reply:
So this is the pattern across the related Classes and I'd like to leave it for consistency. I can't recall off the top of my head, but doesn't the dunder unicode also offer some compatibility in some circumstances?

Reply:
The Meta can be deleted, though.

Reply:
Actually, can you confirm unique_together does not apply here, if you look at the Title class below this, for example?

Reply:
@sblack-usu can you confirm this?

Reply:
The update method is just calling the parent method, which will continue if it's removed.  The remove method is duplicating the same logic of the parent method.  I see no reason to leave them unless there is intent to modify the behavior.

unicode should stay, basically for casting the object to a string.

unique_together wouldn't hurt, but I don't think it really buys us anything.  We only allow one citation, so it shouldn't be repeatable by that constraint.